widgets/DynamoDB-batchsink.json

```diff
@@ -0,0 +1,98 @@
+{
+  "metadata": {
+    "spec-version": "1.0"
+  },
+  "configuration-groups": [
+    {
+      "label": "DynamoDB Configuration",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Reference Name",
+          "name": "referenceName"
+        },
+        {
+          "widget-type": "textbox",
```

Comment:
We had kept this assuming cases where the user is running DynamoDB in local mode for testing purpose. In this case, providing only the option for region id may not be enough. Also, providing a drop-down of options will not feasible for such cases as user can have the endpointURl as localhost:<anyAvailablePort>.
The current implementation allows the user to skip regionId or endpointURL or both. The plugin will create the regionId or endpointURL based on the other parameters provided(accessKey, and secretAccessKey).
Please, let me know if we still need to remove the endpointURL or take it as a drop-down widget instead of a text-box.